public/services/datasources/datasource_services/types/datasource_service.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface IDatasourceService {
+  getInstance: (dsRef: any) => any;
+  getList: (filters: any) => any;
```

Comment:
np. possible to reduce the `any`s?..

Reply:
yea like others, will avoid any all over the places, i've got more changes coming in for this, for now it's just a placeholder